src/extensions/mvs/tree/mvs/mvs-primitives.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2023-2024 mol* contributors, licensed under MIT, See LICENSE file for more info.
+ *
+ * @author David Sehnal <david.sehnal@gmail.com>
+ */
+
+import { float, int, literal, mapping, nullable, obj, str, union, ValueFor } from '../generic/params-schema';
+import type { MVSNode } from './mvs-tree';
+import { ColorT, FloatList, IntList, PrimitivePositionT, StrList } from './param-types';
+
+// TODO: Figure out validation and default values for these
+
+const _LineBase = {
+    start: PrimitivePositionT,
+    end: PrimitivePositionT,
+    thickness: nullable(float),
+    color: nullable(ColorT),
+    dash_start: nullable(float),
+    dash_length: nullable(float),
+    gap_length: nullable(float),
```

Comment:
Is there a default value for `gap_length` which is used when e.g. `dash_length` is specified, while the `gap_length` is not?

Reply:
This is what we will have to figure out separately how to effectively handle this with @midlik .